services/frontendhost/frontend.go

```diff
@@ -1301,12 +1301,25 @@ func (h *Frontend) GetQueueDepthInfo(ctx thrift.Context, queueRequest *c.GetQueu
 	if _, err = h.prolog(ctx, queueRequest); err != nil {
 		return
 	}
+	var cgDesc *c.ConsumerGroupDescription
 
-	cgUUID := queueRequest.GetKey()
+	cgUUID := queueRequest.GetConsumerGroupName()
```

Comment:
This is similar to other APIs, where ConsumerGroupName can be the name or (maybe) the UUID. This naming is consistent with the 95% use case of specifying destination path and name, and consistent with the 5% case of overloading it with the UUID, similar to other Frontend APIs